docs/source/pages/howToWorkWithPlatform/mainApplication/templateList/templateManagement/templateCreationEdition/processingFlowOnTemplate.rst

```diff
@@ -76,11 +76,29 @@ In AUTO flow system configure processing flow by the following rules:
 
 3. CC is added after last EDS Signer (if EDS Signer has been already added to the flow) or after last Assignee (if Assignee has been already added to the flow and EDS Singer was not added), or after Sender if no role except Sender was added.
 
-MANUAL flow
-===========
+Consistent flow
+===============
+
+Logic of addition roles to the flow similar to AUTO flow, but user can change the order. To change the order user has to enable "Set ordering process" toggle and set order using drag and drop option for roles or edit order number of each role manually. For example you can create a flow where signer role can be before assignee role in case when assignee has no actions in current document of the template and has to upload other documents.
+
+Some restrictions of the role places in the flow:
+
+1. Assignee cannot have order in the flow higher than first EDS Signer
+
+2. EDS signer cannot have order lower than last Assignee
+
+3. Copy can have any order in the flow but not less than 1
+
+.. warning:: **If you have an external documets with e-ink signatures in your template you will have to place document uploader roles before roles which are signing these documents in MANUAL mode. Order other then that wil not be allowed by the application.**
```

Comment:
wil should be will.

Also I can't understand this sentence: "Order other then that wil not be allowed by the application" can you please explain it to me in chat?